app/assets/stylesheets/michigan-benefits/administrate-extensions.scss

```diff
@@ -0,0 +1,14 @@
+.main-content__funnel {
```

Comment:
I remember you saying that you struggled to figure out how to do this w/administrate  -- thoughts on linking to relevant docs as a comment at the top of this file?

Reply:
Sadly it wasn't actually documented, I had to dig into the source code to find it :(

Reply:
even better! if you can [permalink](https://help.github.com/articles/getting-permanent-links-to-files/) to the source code that you found, that would be awesome